config.toml.example

```diff
@@ -171,37 +176,37 @@ changelog-seen = 2
 # first compiler.
 #
 # Defaults to host platform
-#build = "x86_64-unknown-linux-gnu"
+#build = <host platform>
 
 # Which triples to produce a compiler toolchain for. Each of these triples will
 # be bootstrapped from the build triple themselves.
 #
 # Defaults to just the build triple
-#host = ["x86_64-unknown-linux-gnu"]
+#host = [build]
 
 # Which triples to build libraries (core/alloc/std/test/proc_macro) for. Each of
 # these triples will be bootstrapped from the build triple themselves.
 #
 # Defaults to `host`. If you set this explicitly, you likely want to add all
 # host triples to this list as well in order for those host toolchains to be
 # able to compile programs for their native target.
-#target = ["x86_64-unknown-linux-gnu"]
+#target = host
```

Comment:
At a glance this makes it seem like it doesn't accept an array I think.

Reply:
I changed this to `#target = host (array)` - do you think that makes it more clear?

Reply:
I think so.